File: gclib/src/main/java/com/github/guqt178/utils/log/ALog.java
package com.github.guqt178.utils.log;

import org.jetbrains.annotations.NotNull;

/**
 * xLog
 * com.elvishew.xlog.XLog
 */
public class ALog {

    public static void wtf(@NotNull String msg, @NotNull Object... format) {
        LoggerWrapper.getLogger().wtf(msg, format);
    }


    public static void warn(@NotNull String msg, @NotNull Object... format) {
        LoggerWrapper.getLogger().warn(msg, format);

    }


    public static void info(@NotNull String msg, @NotNull Object... format) {
        LoggerWrapper.getLogger().info(msg, format);

    }


    public static void error(@NotNull String msg, @NotNull Object... format) {
        LoggerWrapper.getLogger().error(msg, format);

    }


    public static void debug(@NotNull String msg, @NotNull Object... format) {
        LoggerWrapper.getLogger().debug(msg, format);

    }


    public static void tag(@NotNull String tag) {
        LoggerWrapper.getLogger().tag(tag);

    }

    @NotNull

    public static String getTag() {
        return LoggerWrapper.getLogger().getTag();
    }
}

File: libs/gclib/src/main/kotlin/com/github/guqt178/utils/log/ALog.kt
package com.github.guqt178.utils.log

import com.github.guqt178.utils.log.LoggerWrapper.Companion.logger

/**
 * xLog
 * com.elvishew.xlog.XLog
 */
object ALog {
    fun wtf(msg: String, vararg format: Any) {
        logger!!.wtf(msg, *format)
    }

    fun warn(msg: String, vararg format: Any) {
        logger!!.warn(msg, *format)
    }

    fun info(msg: String, vararg format: Any) {
        logger!!.info(msg, *format)
    }

    fun error(msg: String, vararg format: Any) {
        logger!!.error(msg, *format)
    }

    fun debug(msg: String, vararg format: Any) {
        logger!!.debug(msg, *format)
    }

    fun tag(tag: String) {
        logger!!.tag(tag)
    }

    val tag: String
        get() = logger!!.getTag()
}